curated_transformers/layers/activations.py

```diff
@@ -71,6 +32,14 @@ class Activation(Enum, metaclass=_ActivationMeta):
     #: Sigmoid Linear Unit (`Hendrycks et al., 2016`_).
     SiLU = "silu"
 
+    @classmethod
+    def _missing_(cls, value):
```

Comment:
I actually think this is more elegant than how we had it before :-)

Reply:
Yeah, somehow I missed (maybe pun intended?) this before.